webpack.config.js

```diff
@@ -50,6 +50,7 @@ export default {
       fileURLToPath(new URL('web_src/js/index.js', import.meta.url)),
       fileURLToPath(new URL('node_modules/easymde/dist/easymde.min.css', import.meta.url)),
       fileURLToPath(new URL('web_src/fomantic/build/semantic.css', import.meta.url)),
+      fileURLToPath(new URL('node_modules/katex/dist/katex.css', import.meta.url)),
```

Comment:
This file includes a lot of font resources. We should not unconditionally load those on every page. Remove this line and change the JS import to 

```js
const [{default: katex}] = await Promise.all([
  import(/* webpackChunkName: "katex" */'katex');
  import(/* webpackChunkName: "katex" */'katex//dist/katex.css');
]);
```
which will then lazy-load the CSS and fonts as well.